modules/videopress/shortcode.php

```diff
@@ -129,6 +129,9 @@ function videopress_shortcode_callback( $attr ) {
  */
 wp_oembed_add_provider( '#^https?://videopress.com/v/.*#', 'http://public-api.wordpress.com/oembed/1.0/', true );
 
+// Register a VideoPress handler for direct links to .mov files (and potential other non-handled types later).
+wp_embed_register_handler( 'video', '#^https?://videos.(videopress.com|files.wordpress.com)/.+?\.(mov)$#i', apply_filters( 'wp_video_embed_handler', 'wp_embed_handler_video' ), 10 );
```

Comment:
Do you think you could break this into separate lines, so we can add the following docblock before `apply_filters` to please our parser?

`/** This filter is already documented in core/wp-includes/embed.php */`